server/dao/listing.js

```diff
@@ -51,6 +51,11 @@ class ListingDao {
                 ELSE rentr_listing.title END
                 FROM rentr_listing
                 WHERE id = $10),
+        userid = (SELECT
+                CASE WHEN userid = $11 IS NOT NULL THEN $11 
+                ELSE rentr_listing.userid END
+                FROM rentr_listing
+                WHERE id = $10),
```

Comment:
1. Is `id` refers to `userid` and `lid` refers to `listingid`? If yes, the where-clause needs to be updated: `WHERE id = $11`. (Personally I would prefer `id` to be listingid and `uid` to be userid.)
2. Within each listing, user should not be able to modify `userid` associated to the listing. Can we replace this userid section with simply `userid=$10` instead?

Reply:
For (1), it's other way around. I know it's confusing, I will fix that.
For (2), I can do that.

Reply:
Sounds good! Thanks, Aziz!